src/Protobuf/Decode.elm

```diff
@@ -27,7 +27,7 @@ values.
 
 # Integers
 
-@docs int32, uint32, sint32, fixed32, sfixed32
+@docs int32, uint32, sint32, fixed32, sfixed32, int64, uint64, sint64, fixed64, sfixed64, intType, uintType, sintType, fixed64Type
```

Comment:
remove `intType, uintType, sintType, fixed64Type`

Reply:
I thought you wanted to expose them so consumers could use other types than the built-in basic Int64 type.
Although you could always convert from Int64 to your custom type via `getInt32s`.

Reply:
Yes I originally wanted that, but if we're also providing the logic there is no need for that